docusaurus.config.js

```diff
@@ -51,6 +51,21 @@ const config = {
         keywords: { boost: 10 },
       },
     }],
+    () => ({
+      name: "custom-yaml-loader",
+      configureWebpack() {
+        return {
+          module: {
+            rules: [
+              {
+                test: /\.ya?ml$/,
+                use: "yaml-loader",
```

Comment:
This adds an inline plugin, which changes Docusaurus's Webpack config. Webpack is the "bundler" (ie. compiler) that handles turning your 500mb of node_modules (libraries) into a minimal set of js for the browser to consume. 

It adds a new `yaml-loader` library I found, which transforms any .yml or .yaml files that get imported, into Javascript objects.